.github/workflows/build-wheels-pypi.yml

```diff
@@ -0,0 +1,120 @@
+name: Build Cross platform / architecture wheels and upload to PyPi
+# See https://github.com/pypa/cibuildwheel for details
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "test[0-9]+.[0-9]+.[0-9]+"
+
+  # Trigger on request.
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: Build wheel for ${{ matrix.os }}, Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, macos-latest, windows-2022]
+        python-version: ["3.9", 3.10",  "3.11"]
```

Comment:
I thought we were going to support `3.8` as well?

Reply:
We are 3.8 compatible, but I considered not building wheels for it.
If you think 3.8 distribution is important, I'll add in back in

Reply:
I'm cool with tht

Reply:
its is agreed and documented that will build against the 3 most recent CPython versions